interceptors/logging/interceptors_test.go

```diff
@@ -9,18 +9,18 @@ import (
 	"sync"
 	"testing"
 
-	middleware "github.com/grpc-ecosystem/go-grpc-middleware"
-	"github.com/grpc-ecosystem/go-grpc-middleware/grpctesting"
-	"github.com/grpc-ecosystem/go-grpc-middleware/interceptors"
-	"github.com/grpc-ecosystem/go-grpc-middleware/interceptors/logging"
-	"github.com/grpc-ecosystem/go-grpc-middleware/interceptors/tags"
+	middleware "github.com/grpc-ecosystem/go-grpc-middleware/v2"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/logging"
+	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors/tags"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/codes"
 
-	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/grpctesting/testproto"
+	pb_testproto "github.com/grpc-ecosystem/go-grpc-middleware/v2/grpctesting/testproto"
```

Comment:
What import grouping do we want to use? This is a little inconsistent. I usually do 3 import groups. Maybe now is a good time to change?

Reply:
so I use goimports which does 2 groups. What do you prefer? I am fine with whatever as long as there is popular auto formatting tool for this (: